playbooks/roles/hammer-devel/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+hammer_ruby_version: 2.3.1
+hammer_host: 'http://katello-devel.example.com:3000/'
+hammer_username: admin
+hammer_password: changeme
```

Comment:
Best practices encourage prefix with the role namespace, in other words, hammer_devel vs hammer